packages/doenetml-worker-rust/lib-doenetml-core/src/core/doenet_ml_core.rs

```diff
@@ -409,7 +409,7 @@ impl DoenetMLCore {
     ///
     /// Include warnings as a separate vector (errors are embedded in the tree as elements).
     pub fn to_flat_dast(&mut self) -> FlatDastRoot {
-        //log!("***Calling to flat dast on core****");
+        log!("***Calling to flat dast on core****");
```

Comment:
Great, I'm glad we can have logs again!

You can delete this `log!` now, if you like, as I verified `to_flat_dast` is not called twice if I run the built version of test-viewer.

Reply:
Removed!